.github/workflows/check.yml

```diff
@@ -37,6 +37,7 @@ jobs:
         run: yarn test
 
   migrations:
+    if: github.event_name == 'pull_request'
```

Comment:
Will this run it in a PR from a fork too?

Reply:
I believe so, `pull_request` events are emitted by both in-repo and fork PRs.